README.md

```
@@ -21,55 +21,82 @@ The QC metrics calculated by Picard tools and output by this app are informative
+    - See https://gatk.broadinstitute.org/hc/en-us/articles/360037225812-CollectTargetedPcrMetrics-Picard-
 
-**bedfile**:
-A BED file defining the enriched regions.
+The following files are required for RNA-seq QC analysis to run:
 
-**refflat_file**:
-A refflat file containing reference transcripts.
+- **sorted_bam**
+    - A coordinate-sorted mapping file in BAM format (`*.bam`). BAM files generated by commonly used mappers such as BWA, BWA-MEM, Bowtie, TopHat, HISAT and NovoAlign are acceptable as input. 
+- **refflat_file**:
+    - A refflat file containing reference transcripts.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)